src/controllers/admin/office.ts

```diff
@@ -42,3 +44,155 @@ export const AdminLoginHandler = async (req: Request, res: Response) => {
     });
   }
 }
+
+export const AdminDashboardHandler = async (req: Request, res: Response) => {
+  try{
+    const activeMembers_thisyear = await prismaClient.user.count({
+      where: {
+        createdAt : {gte : new Date(new Date().getFullYear(),0,1)}
+      }
+    });
+
+    const activeMembers_alltime = await prismaClient.user.count()
+
+    const activeMembers_lastyear = await prismaClient.user.count({
+      where: {
+        createdAt : {gte : new Date(new Date().getFullYear()-1,0,1), lt : new Date(new Date().getFullYear(),0,1)}
+      }
+    });
+
+    const EventsCompleted_thisyear = await prismaClient.event.count({
```

Comment:
Any reason for starting the variable name with a `capital` letter? All the others start with a `lowercase` letter. Will be good if we follow a uniform pattern.